x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/data/rule_migrations_data_migration_client.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { v4 as uuidV4 } from 'uuid';
+import type { StoredSiemMigration } from '../types';
+import { RuleMigrationsDataBaseClient } from './rule_migrations_data_base_client';
+
+export class RuleMigrationsDataMigrationClient extends RuleMigrationsDataBaseClient {
+  async create(): Promise<string> {
+    const migrationId = uuidV4();
+    const index = await this.getIndexName();
+    const profileUid = await this.getProfileUid();
+    const createdAt = new Date().toISOString();
+
+    await this.esClient
+      .index({
+        refresh: 'wait_for',
+        id: migrationId,
+        index,
+        document: {
+          created_by: profileUid,
+          created_at: createdAt,
+        },
+        op_type: 'create',
```

Comment:
Is this `op_type: 'create'` the same as using ` await this.esClient.create({...` instead of `.index({...`?

Reply:
Ohh yes, you are right. Fixed in  60e84507f31